Tests/Marketplace/marketplace_services.py

```diff
@@ -1421,6 +1435,29 @@ def format_metadata(self, user_metadata, pack_content_items, integration_images,
         finally:
             return task_status
 
+    def _handle_pack_create_date(self, index_folder_path):
+        """ Gets the pack created date.
+        Args:
+            index_folder_path (str): downloaded index folder directory path.
+        Returns:
+            datetime: Pack created date.
+
+        """
+
+        # load changelog from downloaded index
+        changelog_index_path = os.path.join(index_folder_path, self._pack_name, Pack.CHANGELOG_JSON)
+        changelog = {}
+        if os.path.exists(changelog_index_path):
+            try:
+                with open(changelog_index_path, "r") as changelog_file:
+                    changelog = json.load(changelog_file)
+            except json.JSONDecodeError:
+                changelog = {}
+        init_changelog_version = changelog.get(Pack.PACK_INITIAL_VERSION, {})
```

Comment:
Does this name truly represent what's in this variable?

Reply:
I think it is, this var is for the first version in the changelog file ( 1.0.0)
changed to `initial_changelog_version`